frontend/src/metabase/visualizations/echarts/pie/option/index.ts

```diff
@@ -38,21 +52,57 @@ export function getPieChartOption(
   if (!getColumnSettings) {
     throw Error(`"settings.column" is undefined`);
   }
+  const metricColSettings = getColumnSettings(
+    chartModel.colDescs.metricDesc.column,
+  );
 
+  // "Show total" setting
   const formatMetric = (value: unknown) =>
     renderingContext.formatValue(value, {
-      ...getColumnSettings(chartModel.colDescs.metricDesc.column),
+      ...metricColSettings,
+    });
+
+  const graphicOption = settings["pie.show_total"]
+    ? getTotalGraphicOption(chartModel.total, formatMetric, renderingContext)
+    : undefined;
+
+  // "Show percentages: On the chart" setting
+  const formatPercent = (value: unknown) =>
+    renderingContext.formatValue(value, {
+      column: metricColSettings.column,
+      number_separators: metricColSettings.number_separators,
+      number_style: "percent",
+      jsx: true,
```

Comment:
Seems like it is not emitting `jsx` in this specific case but I assume it should not be true because echarts won't be able to render it anyway

Reply:
Good point, I just copied this from `PieChart.jsx` but you're right it should not be `true` here, I'll fix it.

Reply:
Done